ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -507,7 +514,8 @@ let assert_can_boot_here ~__context ~self ~host ~snapshot ?(do_sr_check=true) ?(
   assert_can_see_networks ~__context ~self ~host;
   if vm_needs_iommu ~__context ~self then
     assert_host_has_iommu ~__context ~host;
-  assert_gpus_available ~__context ~self ~host;
+  if has_non_allocated_vgpus ~__context ~self then
+    assert_gpus_available ~__context ~self ~host;
   assert_usbs_available ~__context ~self ~host;
   assert_netsriov_available ~__context ~self ~host;
```

Comment:
SR-IOV has the same issue.

Reply:
Can you raise a ticket for this?

Reply:
I will raise a ticket for SR-IOV for tracking.

Reply:
Thanks